tests/web/test_transactions.py

```diff
@@ -276,6 +276,7 @@ def test_post_invalid_transfer_transaction_returns_400(b, client, user_pk):
     assert res.json['message'] == expected_error_message
 
 
+@pytest.mark.tendermint
```

Comment:
That's amazing! :100: 
We can now enable tests without changing them because we maintain backward compatibility.